evap/evaluation/models.py

```diff
@@ -262,7 +262,7 @@ def save(self, *args, **kw):
             self.contributions.create(contributor=None)
             del self.general_contribution  # invalidate cached property
 
-        assert self.vote_end_date >= self.vote_end_date
+        assert self.vote_end_date >= self.vote_start_datetime.date()
```

Comment:
i would prefer such a change to be in a separate commit since it has little to do with the feature you implemented, but you don't need to change it for this PR.